src/hex/vertex.rs

```diff
@@ -217,6 +313,55 @@ mod tests {
         );
     }
 
+    #[test]
+    fn adjacent_edges() {
+        assert_eq!(
+            vertex!(0, 0, VertexSpin::Up).adjacent_edges(),
+            [
+                edge!(1, -1, EdgeDirection::West),
+                edge!(0, 0, EdgeDirection::NorthEast),
+                edge!(0, 0, EdgeDirection::NorthWest),
+            ]
+        );
+
+        assert_eq!(
+            vertex!(0, 0, VertexSpin::Down).adjacent_edges(),
+            [
+                edge!(0, 1, EdgeDirection::NorthWest),
+                edge!(0, 1, EdgeDirection::West),
+                edge!(-1, 1, EdgeDirection::NorthEast),
+            ]
+        );
+    }
+
+    #[test]
+    fn distance() {
```

Comment:
Do we need tests for distance between non-adjacent vertices?

Reply:
Added 2 more just in case but they should be good.